test/e2e/cli/mongocli/mongocli.go

```diff
@@ -72,6 +72,11 @@ func DeleteCluster(projectID, clusterName string) *Buffer {
 	return session.Wait().Out
 }
 
+func DeleteProject(projectID string) *Buffer {
```

Comment:
Do you still need this function?

Reply:
Thanks... removed -> https://github.com/commercetools/mongodb-atlas-kubernetes/pull/1/commits/22f0f0d0f16ae2d7c11b9b981f6b8b4946847c53